spec/models/program_spec.rb

```diff
@@ -4,6 +4,8 @@
   subject { create(:program) }
   let!(:conference) { create(:conference, end_date: Date.today + 3) }
   let!(:program) { conference.program }
+  let!(:event_program) { create(:program, selected_schedule: schedule)}
```

Comment:
We don't really need an event_program just for testing the events. I will use program